packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -256,3 +256,17 @@ function sanitizeTiming(name: string) {
   }
   return sanitized
 }
+
+function cloneLocation(location: Location): ReadonlyLocation {
```

Comment:
I'm wondering if "clone" has the right meaning in this case.

Reply:
What do you think about `createReadonlyLocation` then?

Reply:
yes, nice